standard/OAPI_Coverages.adoc

```diff
@@ -1,14 +1,19 @@
-:Title: OGC API Coverages
-:titletext: OGC API Coverages
+:Title: OGC API-Coverages
```

Comment:
OAPI - Features uses "OGC API - Features - Part 1: Core", should we harmonize?

Reply:
@Schpidi I think this is an issue for the Doc Team or Naming Authority.  Personally I have a problem calling it "core" when it includes more than just the Core Requirements Class.

Reply:
Telco 20191218: Agreed to use "Part 1: Core" also here.